src/App.tsx

```diff
@@ -17,18 +59,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onSetQuery={setQuery}
+                onChangeFilter={setFilterBy}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && !isError && (
+                <Loader />
+              )}
+
+              <TodoList
+                todos={visibleTodos}
+                onSelectedTodo={changeSelectedTodo}
+                selectedTodo={selectedTodoId}
+              />
```

Comment:
In this case, I see a loader and an empty table. Let's show only the loader when the table is empty

![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/84296041/163a18c7-dd20-483e-a771-94db1c1806f9)